pkgs/applications/misc/pomodoro/default.nix

```
@@ -0,0 +1,22 @@
+{ lib, fetchFromGitHub, rustPlatform, Foundation }:
```

```suggestion
{ lib, stdenv, fetchFromGitHub, rustPlatform, Foundation }:
```
somehow forget half of the code